packages/main/src/WheelSlider.js

```diff
@@ -279,28 +270,45 @@ class WheelSlider extends UI5Element {
 	}
 
 	_selectElementByIndex(currentIndex) {
-		const sliderElement = this.shadowRoot.getElementById(`${this._id}--items-list`);
-		const itemsCount = this._itemsToShow.length;
-		const itemCellHeight = this._itemCellHeight ? this._itemCellHeight : 2.875;
-		const offsetStep = isPhone() ? 4 : 2;
 		let index = currentIndex;
+		const itemsCount = this._itemsToShow.length;
+		const scrollBy = 46 * index;
+		const sliderElementsWrapper = this.shadowRoot.querySelector(`#${this._id}--wrapper`);
 
 		if (this.cyclic) {
-			index = this.handleArrayBorderReached(index);
+			index = this._handleArrayBorderReached(index);
 		}
 
 		if (index < itemsCount && index > -1) {
-			const offsetSelectedElement = offsetStep * itemCellHeight - (index * itemCellHeight);
-			sliderElement.setAttribute("style", `top:${offsetSelectedElement}rem`);
+			sliderElementsWrapper.scrollTo(0, scrollBy);
```

Comment:
The change doesn't work on IE: Possible Unhandled Promise Rejection: TypeError: Object doesn't support property or method 'scrollTo'
Most probably it is related to this line. Shouldn't you call scrollTo on the ScrollEnablement instance rathen than on the div?